docs/public-documentation/Output.md

```diff
@@ -77,11 +282,288 @@ For connection to a secure Kafka Broker -
+})
+```
+
+Here's Powershell code that sends multiple messages:
+
+```ps1
+using namespace System.Net
+
+# Input bindings are passed in via param block.
+param($Request, $TriggerMetadata)
+
+# Write to the Azure Functions log stream.
+Write-Host "PowerShell HTTP trigger function processed a request."
+
+$message = @("one", "two")
+
+# Associate values to output bindings by calling 'Push-OutputBinding'.
+Push-OutputBinding -Name outputMessage -Value ($message)
+
+Push-OutputBinding -Name Response -Value ([HttpResponseContext]@{
+    StatusCode = [HttpStatusCode]::OK
+})
+```
+
+### Python
+
+The following example shows a Kafka output binding in a function.json file and a Python function that uses the binding. The function reads in the message from an HTTP trigger and outputs it to the Kafka topic.
+
+Here is the binding data in the _function.json_ file:
+
+```json
+{
+  "scriptFile": "main.py",
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": [
+        "get"
+      ]
+    },
+    {
+      "type": "kafka",
+      "direction": "out",
+      "name": "outputMessage",
+      "brokerList": "BrokerList",
+      "topic": "topic",
+      "username": "%ConfluentCloudUserName%",
+      "password": "%ConfluentCloudPassword%",
+      "protocol": "SASLSSL",
+      "authenticationMode": "PLAIN"
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "$return"
+    }
+  ]
+}
+```
+Here is the Python script code:
+
+In _init_.py:
+```py
+import logging
+
+import azure.functions as func
+
+
+def main(req: func.HttpRequest, outputMessage: func.Out[str]) -> func.HttpResponse:
+    input_msg = req.params.get('message')
+    outputMessage.set(input_msg)
+    return 'OK'
+```
+
+Here's Python code that sends multiple messages:
+
+```py
+def main(req: HttpRequest, outputMessage: Out[str] ) -> HttpResponse:
+    outputMessage.set(['one', 'two'])
+    return 'OK'
+```
+
+### Typescript
+
+The following example shows a Kafka output binding in a function.json file and a Typescript function that uses the binding. The function reads in the message from an HTTP trigger and outputs it to the Kafka topic.
+
+Here is the binding data in the _function.json_ file:
+
+```json
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": [
+        "get"      
+       ]
+    },
+    {
+      "type": "kafka",
+      "name": "outputKafkaMessage",
+      "topic": "topic",
+      "brokerList": "%BrokerList%",
+      "username": "%ConfluentCloudUserName%",
+      "password": "%ConfluentCloudPassword%",
+      "protocol": "SASLSSL",
+      "authenticationMode": "PLAIN",
+      "direction": "out"
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "res"
+    }
+  ],
+  "scriptFile": "../dist/KafkaOutput/index.js"
```

Comment:
If this is autogenerated can we remove this ??

Reply:
I don't think we can remove it because internally, typescript gets compiled to javascript. The core tools would use this compiled javascript file for execution.

Reply:
did you manually added this scriptFile ??

Reply:
It gets autogenerated whenever you create any function using core tools.